######################################################################################################################################################################################################################################################################
##                                                                                                                                                                                                                                                                  ##
##                                                                                                                                                                                                                                                                  ##
##        Copyright (C) 2022 AGNITAS AG (https://www.agnitas.org)                                                                                                                                                                                                   ##
##                                                                                                                                                                                                                                                                  ##
##        This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.    ##
##        This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.           ##
##        You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.                                                                                                            ##
##                                                                                                                                                                                                                                                                  ##
######################################################################################################################################################################################################################################################################
diff -ur slang-1.4.9/src/sltoken.c slang-1.4.9-patched/src/sltoken.c
--- slang-1.4.9/src/sltoken.c	2003-03-23 08:06:40.000000000 +0100
+++ slang-1.4.9-patched/src/sltoken.c	2010-11-16 15:23:57.000000000 +0100
@@ -9,7 +9,7 @@
 #include "slang.h"
 #include "_slang.h"
 
-#define MAX_TOKEN_LEN 254
+#define MAX_TOKEN_LEN 65536
 #define MAX_FILE_LINE_LEN 256
 
 static char Empty_Line[1] = {0};
